### PR TITLE
Contribute KiloM2, CentiM4, and DecaPOISE

### PR DESCRIPTION
**Pull request by fkleedorfer**
Adds those two units plus a number of qudt:hasQuantityKind triples for some units of SpecificHeatCapacity

### CODE REVIEW

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl [hunk 1]
```diff
@@ -31596,3 +31644,40 @@ THE UCUM TABLE (IN ALL FORMATS), UCUM DEFINITIONS, AND SPECIFICATION ARE PROVIDE
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "QUDT Unit of Measure Vocabulary Metadata Version 2.1.35" ;
 .
+
+unit:KiloM2 a qudt:Unit;
```
Comment (steveraysteveray) [inline vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:31648]: This is already present.

**Review by steveraysteveray — CHANGES_REQUESTED** [review]
In reviewing this, I found errors in the dimensionality of:
unit:BTU_IT-PER-LB_F-DEG_F
unit:BTU_IT-PER-LB_F-DEG_R
This was not your doing, but somehow those two slipped by your magic dimensionality algorithm. The dimensionality should be L/H, because LB_F is equivalent to ML/T**2.
Therefore, the referenced dimensionvector and quantitykinds are wrong. Would you mind fixing them? Not sure what the appropriate quantitykind would be, but we could have LinearThermalExpansion as a place holder.

Also, KiloM2 already exists (see embedded comment)

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl (hunk 1 above)
Comment (fkleedorfer) [reply vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl]: rebased onto main and removed the duplicate unit:KiloM2

**Comment by fkleedorfer** [comment]
> In reviewing this, I found errors in the dimensionality of: unit:BTU_IT-PER-LB_F-DEG_F unit:BTU_IT-PER-LB_F-DEG_R 

These units are deprecated and therefore exempt from the checks I am running.

**Comment by fkleedorfer** [comment]
Here is the output if I force checking these units; it agrees with your suggestion (great catch btw!) - I'll make the change.

```
  # unit:BTU_IT-PER-LB_F-DEG_F has incorrect dimension vector:
  # 	http://qudt.org/vocab/dimensionvector/A0E0L2I0M0H-1T-2D0, which should be 
  # 	http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H-1T0D0
  # factor unit tree for unit Btu{IT}/(lbf⋅°F)
  #  ──┬ unit:BTU_IT-PER-LB_F-DEG_F    	http://qudt.org/vocab/dimensionvector/A0E0L2I0M0H-1T-2D0
  #    ├──┬ unit:BTU_IT                   	http://qudt.org/vocab/dimensionvector/A0E0L2I0M1H0T-2D0
  #    │  └──┬ unit:J                        	http://qudt.org/vocab/dimensionvector/A0E0L2I0M1H0T-2D0
  #    │     ├─── unit:M                        	http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H0T0D0
  #    │     └──┬ unit:N                        	http://qudt.org/vocab/dimensionvector/A0E0L1I0M1H0T-2D0
  #    │        ├──┬ unit:KiloGM                   	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T0D0
  #    │        │  └─── unit:GM                       	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T0D0
  #    │        ├─── unit:M                        	http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H0T0D0
  #    │        └─── unit:SEC^-2                   	http://qudt.org/vocab/dimensionvector/A0E0L0I0M0H0T-2D0
  #    ├──┬ unit:LB_F^-1                  	http://qudt.org/vocab/dimensionvector/A0E0L-1I0M-1H0T2D0
  #    │  ├──┬ unit:FT                       	http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H0T0D0
  #    │  │  └──┬ unit:IN                       	http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H0T0D0
  #    │  │     └─── unit:M                        	http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H0T0D0
  #    │  ├─── unit:SEC^-2                   	http://qudt.org/vocab/dimensionvector/A0E0L0I0M0H0T-2D0
  #    │  └──┬ unit:SLUG                     	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T0D0
  #    │     └──┬ unit:LB                       	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T0D0
  #    │        └──┬ unit:KiloGM                   	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T0D0
  #    │           └─── unit:GM                       	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T0D0
  #    └──┬ unit:DEG_F^-1                 	http://qudt.org/vocab/dimensionvector/A0E0L0I0M0H-1T0D0
  #       └─── unit:K                        	http://qudt.org/vocab/dimensionvector/A0E0L0I0M0H1T0D0
  # 
  # the unit has these quantityKinds: SpecificHeatCapacity
  #  ══╤ qk:SpecificHeatCapacity
  #    ╞═══ qk:ExhaustGasesSpecificHeat
  #    ├─── unit:BTU_IT-PER-LB-DEG_F
  #    ├─── unit:BTU_IT-PER-LB-DEG_R
  #    ├─── unit:BTU_IT-PER-LB_F-DEG_F
  #    ├─── unit:BTU_IT-PER-LB_F-DEG_R
  #    ├─── unit:BTU_TH-PER-LB-DEG_F
  #    ├─── unit:CAL_IT-PER-GM-DEG_C
  #    ├─── unit:CAL_IT-PER-GM-K
  #    ├─── unit:CAL_TH-PER-GM-DEG_C
  #    ├─── unit:CAL_TH-PER-GM-K
  #    ├─── unit:J-PER-GM-DEG_C
  #    ├─── unit:J-PER-GM-K
  #    ├─── unit:J-PER-KiloGM-DEG_C
  #    ├─── unit:J-PER-KiloGM-K
  #    ├─── unit:KiloCAL-PER-GM-DEG_C
  #    └─── unit:M2-PER-SEC2-K
  # quantity kinds that fit the corrected dimension vector http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H-1T0D0, and associated units:
  #  ══╤ qk:LinearThermalExpansion
  #    ├─── unit:CentiM-PER-K
  #    ├─── unit:FT-PER-DEG_F
  #    ├─── unit:IN-PER-DEG_F
  #    ├─── unit:M-PER-K
  #    ├─── unit:MicroM-PER-K
  #    ├─── unit:MilliM-PER-K
  #    └─── unit:YD-PER-DEG_F
unit:BTU_IT-PER-LB_F-DEG_F qudt:hasDimensionVector qkdv:A0E0L1I0M0H-1T0D0 .

  # unit:BTU_IT-PER-LB_F-DEG_R has incorrect dimension vector:
  # 	http://qudt.org/vocab/dimensionvector/A0E0L2I0M0H-1T-2D0, which should be 
  # 	http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H-1T0D0
  # factor unit tree for unit Btu{IT}/(lbf⋅°R)
  #  ──┬ unit:BTU_IT-PER-LB_F-DEG_R    	http://qudt.org/vocab/dimensionvector/A0E0L2I0M0H-1T-2D0
  #    ├──┬ unit:BTU_IT                   	http://qudt.org/vocab/dimensionvector/A0E0L2I0M1H0T-2D0
  #    │  └──┬ unit:J                        	http://qudt.org/vocab/dimensionvector/A0E0L2I0M1H0T-2D0
  #    │     ├─── unit:M                        	http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H0T0D0
  #    │     └──┬ unit:N                        	http://qudt.org/vocab/dimensionvector/A0E0L1I0M1H0T-2D0
  #    │        ├──┬ unit:KiloGM                   	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T0D0
  #    │        │  └─── unit:GM                       	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T0D0
  #    │        ├─── unit:M                        	http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H0T0D0
  #    │        └─── unit:SEC^-2                   	http://qudt.org/vocab/dimensionvector/A0E0L0I0M0H0T-2D0
  #    ├──┬ unit:LB_F^-1                  	http://qudt.org/vocab/dimensionvector/A0E0L-1I0M-1H0T2D0
  #    │  ├──┬ unit:FT                       	http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H0T0D0
  #    │  │  └──┬ unit:IN                       	http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H0T0D0
  #    │  │     └─── unit:M                        	http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H0T0D0
  #    │  ├─── unit:SEC^-2                   	http://qudt.org/vocab/dimensionvector/A0E0L0I0M0H0T-2D0
  #    │  └──┬ unit:SLUG                     	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T0D0
  #    │     └──┬ unit:LB                       	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T0D0
  #    │        └──┬ unit:KiloGM                   	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T0D0
  #    │           └─── unit:GM                       	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T0D0
  #    └──┬ unit:DEG_R^-1                 	http://qudt.org/vocab/dimensionvector/A0E0L0I0M0H-1T0D0
  #       └─── unit:K                        	http://qudt.org/vocab/dimensionvector/A0E0L0I0M0H1T0D0
  # 
  # the unit has these quantityKinds: SpecificHeatCapacity
  #  ══╤ qk:SpecificHeatCapacity
  #    ╞═══ qk:ExhaustGasesSpecificHeat
  #    ├─── unit:BTU_IT-PER-LB-DEG_F
  #    ├─── unit:BTU_IT-PER-LB-DEG_R
  #    ├─── unit:BTU_IT-PER-LB_F-DEG_F
  #    ├─── unit:BTU_IT-PER-LB_F-DEG_R
  #    ├─── unit:BTU_TH-PER-LB-DEG_F
  #    ├─── unit:CAL_IT-PER-GM-DEG_C
  #    ├─── unit:CAL_IT-PER-GM-K
  #    ├─── unit:CAL_TH-PER-GM-DEG_C
  #    ├─── unit:CAL_TH-PER-GM-K
  #    ├─── unit:J-PER-GM-DEG_C
  #    ├─── unit:J-PER-GM-K
  #    ├─── unit:J-PER-KiloGM-DEG_C
  #    ├─── unit:J-PER-KiloGM-K
  #    ├─── unit:KiloCAL-PER-GM-DEG_C
  #    └─── unit:M2-PER-SEC2-K
  # quantity kinds that fit the corrected dimension vector http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H-1T0D0, and associated units:
  #  ══╤ qk:LinearThermalExpansion
  #    ├─── unit:CentiM-PER-K
  #    ├─── unit:FT-PER-DEG_F
  #    ├─── unit:IN-PER-DEG_F
  #    ├─── unit:M-PER-K
  #    ├─── unit:MicroM-PER-K
  #    ├─── unit:MilliM-PER-K
  #    └─── unit:YD-PER-DEG_F
unit:BTU_IT-PER-LB_F-DEG_R qudt:hasDimensionVector qkdv:A0E0L1I0M0H-1T0D0 .
```

**Comment by fkleedorfer** [comment]
done

**Review by steveraysteveray — APPROVED** [review]

**Pull request merged**